clang/include/clang/Driver/Options.td

```diff
@@ -3494,6 +3494,9 @@ def fno_struct_path_tbaa : Flag<["-"], "fno-struct-path-tbaa">, Group<f_Group>;
 def fno_strict_enums : Flag<["-"], "fno-strict-enums">, Group<f_Group>;
 def fno_strict_overflow : Flag<["-"], "fno-strict-overflow">, Group<f_Group>,
   Visibility<[ClangOption, FlangOption]>;
+def fno_zero_init_global_without_init : Flag<["-"], "fno-zero-init-global-without-init">, Group<f_Group>,
```

Comment:
Following https://github.com/llvm/llvm-project/pull/121619#discussion_r1906391568, I suggest considering `-fskip-zero-init-globals` or similar.

Reply:
This looks like regular boolean flag which should support `clang -fflag -fno-flag -fflag -fno-flag -fflag -fno-flag` invocation. Id rather add positive version for consistency.

There have non-boolean settings https://github.com/llvm/llvm-project/pull/121619#discussion_r1906391568 
we just have no idea what could be a meaning difference for `-fno-sanitize-top-hot=null=0.5` vs `-fsanitize-top-hot=null=0.5`.

Reply:
Added a positive version as well.